app/controllers/v1/doctors_controller.rb

```diff
@@ -17,12 +18,16 @@ def create
   def destroy
     @doctor = Doctor.find(params[:id])
     if @doctor.destroy
-      render json: { status: 201, message: 'doctor has deleted successfully!', content: { doctor: @doctor } }
+      render json: { status: 201, message: 'doctor has been deleted successfully!', content: { doctor: @doctor } }
     else
       render json: { error: 401, message: ' Operation did not succeed!' }
     end
   end
 
+  def doctors_list
+    render json: Doctor.all
+  end
+
```

Comment:
doctor_list action is same to the index action (render json: Doctor.all)

Reply:
Check it well please, they look like they are the same but they are quite different.